net-irc/znc-clientbuffer/znc-clientbuffer-1.0.48.ebuild

```diff
@@ -18,20 +20,18 @@ RDEPEND="${DEPEND}"
 
 DOCS=( README.md )
```

Comment:
You can drop this, as it's the default, when not being set.

Reply:
done